local-cli/generator-android/templates/src/app/proguard-rules.pro

```diff
@@ -40,10 +40,13 @@
 
 -keep class * extends com.facebook.react.bridge.JavaScriptModule { *; }
 -keep class * extends com.facebook.react.bridge.NativeModule { *; }
+-keep class * { native <methods>; }
```

Comment:
I read the code decompiled from classes-proguard. All other methods is find but getType is missing. I don't know why...